vm/src/stdlib/array.rs

```diff
@@ -417,6 +402,9 @@ def_array_enum!(
     (Double, f64, 'd', "d"),
 );
 
+#[derive(Copy, Clone, Ord, PartialOrd, Eq, PartialEq, Debug)]
+pub struct WideChar(libc::wchar_t);
```

Comment:
WASM seems to have no `libc::wchar_t`.

Should I add a cfg like this?
```Rust
#[cfg(target_arch = "wasm32")]
pub struct WideChar(u32);
```

Reply:
It looks good